flocker/provision/_rackspace.py

```diff
@@ -15,6 +16,14 @@ def provision_rackspace(node, package_source, distribution):
     """
     Provision flocker on this node.
     """
+    if distribution in ('centos-7',):
+        run(
+            username='root',
+            address=node.address,
+            commands=task_upgrade_kernel_centos(),
```

Comment:
This seems to be the Rackspace-specific provisioner.  Does the CentOS 7 kernel on the other platforms not need to be upgraded?  The docs seem to suggest the upgrade is required regardless of cloud provider.